modules/lvms-about-scaling-storage-of-clusters.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="lvms-about-scaling-storage-of-cluster_{context}"]
+= Ways to scale up the storage of clusters
+
+{product-title} supports additional worker nodes for {sno} clusters on bare metal user-provisioned infrastructure. You can scale up the storage of clusters either by adding new worker nodes with available storage or by adding new devices to the existing worker nodes. 
```

Comment:
I think multiple nodes are 4.15 and above only:
https://issues.redhat.com/browse/OCPSTRAT-390

That would mean we only run SNO on 4.14, not SNO+workers. @suleymanakbas91 feel free to correct me here if necessary.

Reply:
Confirmed: SNO+workers would be multi-node, which is **only** supported starting V415. So "by adding new worker nodes" is available only starting V4.15. In previous versions, only "by adding new devices to the existing node" is feasable with <=4.14

Reply:
Hi @DanielFroehlich 
Thanks for the confirmation.

I came across a link to this module "[Adding worker nodes to SNO](https://75066--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-sno-worker-nodes)".

The above section has a note saying:
"Unlike multi-node clusters, by default all ingress traffic is routed to the single control-plane node, even after adding additional worker nodes."

So would this make SNO+worker nodes different from multi-node?

If "Adding additional worker nodes to SNO" is supported with <= 4.14, then would "scaling up the storage capacity of the worker nodes" also be supported on SNO if user has added additional worker nodes?

Reply:
Updated as per the suggestion.